features/addressing/managing_countries/editing_country_with_provinces.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country with provinces
+    In order to add or remove provinces in existing countries
+    As an Administrator
+    I want to be able to edit a country and its provinces
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has country "United Kingdom"
+
+    @ui @javascript
+    Scenario: Adding province to existing country
+        Given I want to edit this country
```

Comment:
Please have a look at for instance the already merged `editing_tax_category.feature`, `editing_tax_rate.feature` or `editing_locale.feature`. All of the use the convention I've suggested.